pkg/controller/controlplane/valuesprovider.go

```diff
@@ -600,8 +601,16 @@ func (vp *valuesProvider) getControlPlaneShootChartValues(ctx context.Context, c
 		"enabled": vp.controllerConfig.Storage.Duros.Enabled,
 	}
 
+	// resolve ip address of the apiserver from "api"+Shoot.Spec.DNS.Domain
+	apiserver := fmt.Sprintf("api.%s", *cluster.Shoot.Spec.DNS.Domain)
+	apiserverIPs, err := net.LookupHost(apiserver)
```

Comment:
If there is an error talking to the upstream dns (cloudflare in our case) we could probably look also into dnsentries to make this lookup more robust.

Reply:
This is  created in the shoot in the firewall namespace, but we need another on in the seed for ingress to match this request.

Reply:
> This is created in the shoot in the firewall namespace, but we need another on in the seed for ingress to match this request.

cwnp for seed is not needed, as the ingress-cwnp will be automatically updated by the firewall-controller as soon as the apiserver-service in the seed gets the konnectivity-server port added.

Reply:
> If there is an error talking to the upstream dns (cloudflare in our case) we could probably look also into dnsentries to make this lookup more robust.

SGTM ... will check. Can probably even completely replace the LookupHost(), since we already experienced some "dns lag" after creating dns-entries for newly created apiserver DNS-entries